docker-compose.prod.yml

```diff
@@ -0,0 +1,158 @@
+version: "3.3"
+
+services:
+
+  mysql-opencr:
+    image: mysql:5.7.34
+    container_name: mysql-opencr
+    env_file:
+      - ./opencr/db.env
+    ports:
+      - "3309:3306"
+    volumes:
+      - ./opencr/my.cnf:/etc/mysql/my.cnf
+      - opencrDbData:/var/lib/mysql
+    restart: unless-stopped
+
+  hapi-fhir:
+    image: hapiproject/hapi:v5.4.1
+    container_name: hapi-fhir
+    depends_on:
+      - mysql-opencr
+    ports:
+      - "8181:8080"
+    environment:
+      spring.config.location: 'file:///user/share/hapi-fhir/application.yaml'
+    volumes:
+      - ./opencr/application.yaml:/user/share/hapi-fhir/application.yaml
+    restart: unless-stopped
```

Comment:
I think auto container restart should be avoided, if the application shuts down due to an error, you should only restart the container after resolving it otherwise you will end up with many failures due to premature restarts